pkg/ebpf/ingress_node_firewall_loader.go

```diff
@@ -238,19 +244,25 @@ func (infc *IngNodeFwController) IngressNodeFwAttach(ifacesName []string, isDele
 	return ifMap, nil
 }
 
-// cleanup will delete all link objects for all interfaces and remove all the maps
+// cleanup will delete interface's eBPF objects.
 func (infc *IngNodeFwController) cleanup(ifName string) error {
 	l, ok := infc.links[ifName]
```

Comment:
I just realized that there's a problem with this here:
a) we must hold an instance of infc in memory for the links[] to be populated, otherwise we get: "failed to find Link object for interface"  (I can solve that by creating a long-living instance of infc)
b) while I can easily add a single infc the ebpfsyncer singleton's state in my PR, and while I can solve that issue then for the same process, if the process crashes and then restarts and calls a cleanup for the interface, this will still fail

a) is easy to fix, b) not so much

Reply:
yeah we need to keep infc instance to be able to handle process cleanup and call the new api `CleaneBPFObjs`